src/dmd/dscope.d

```diff
@@ -673,6 +673,8 @@ struct Scope
             tok = TOK.false_;
         else if (ident == Id.unsigned)
             tok = TOK.uns32;
+        else if (ident == Id.wchar_t)
+            tok = global.params.isWindows ? TOK.wchar_ : TOK.dchar_;
```

Comment:
This doesn't appear to have any coverage.

Reply:
That's because it'll produce different results on windows and non-windows machines.

Reply:
There are ways to limit tests in the test suite to specific platforms, so you could add a Windows test and non-Windows test to the test suite.  See the `DISABLED` qualifier e.g. https://github.com/dlang/dmd/blob/15c8da123eddff3b3113349cc968290532d9a9a3/test/runnable/test6423.d#L1

Reply:
Does this make `wchar_t` part of the language? Even `size_t` is just defined in object.d. What about name mangling?

Reply:
> Does this make wchar_t part of the language?

No. The error message happens only if it is an undefined identifier, i.e. is already an error. The idea is to produce better error messages when someone tries to compile a C file with the D compiler.

Reply:
Ah, sorry, only saw the diff and failed to look at the wider context.

Reply:
It's covered for me:

![image](https://user-images.githubusercontent.com/4370550/37884207-1c250e14-30af-11e8-826b-c9a1d0e61e9d.png)


FYI: there seems to be a bug in CodeCov' browser extension that requires a refresh once you switch to the diff page to see the coverage information.
See also: https://github.com/codecov/browser-extension/issues/53